docs/decision-records/README.md

```diff
@@ -0,0 +1,23 @@
+Design decisions about the pipeline are indexed and recorded as individual files in this directory.
+
+To add a new decision, please create a pull request that adds a new markdown file named `XX-short-summary.md` to this directory. When replacing a previous decision, change the status of the latter to "Superseded". The new file should have the following structure:
```

Comment:
To add a new decision, please create a pull request that adds a new markdown file named `XX-short-summary.md` to this directory. When replacing a previous decision, change the status of the latter to "Superseded" and add this to the title of the file `superseded-XX-short-summary.md`. The new file should have the following structure: